csrc/scheduler/tools/loop_domain_scheduler.cpp

```diff
@@ -528,12 +528,10 @@ void scheduleLoopDomainsBy(
     Direction replay_dir_tv = Direction::Undefined;
     if (replay_dir != Direction::Backward &&
         input_ids.size() == transform->inputs().size()) {
-      NVF_ERROR(output_ids.empty());
```

Comment:
I originally thought this should be always the case but that isn't actually case. In particular, there can be a `resize` that produces an output that is mapped with the input. In that case, `output_ids` won't be empty.

Reply:
nitpick: we should remove the comment above on line 525-527 then.